onnxruntime/test/optimizer/nchwc_optimizer_test.cc

```diff
@@ -223,7 +224,7 @@ void NchwcOptimizerTester(const std::function<void(NchwcTestHelper& helper)>& bu
   }
 }
 
-#ifndef DISABLE_CONTRIB_OPS
+#if !defined(DISABLE_CONTRIB_OPS) && !defined(USE_XNNPACK)
```

Comment:
Why are we disabling the tests is USE_XNNPACK is defined? The NchwcTransformer is still enabled in that case.